buildSrc/src/main/kotlin/dev/nolij/zumegradle/JarCompressing.kt

```diff
@@ -306,10 +308,18 @@ open class CompressJarTask : DefaultTask() {
 
 	@TaskAction
 	fun compressJar() {
-		if (useProguard)
-			applyProguard(inputJar, minecraftConfigs, project.rootDir)
-		squishJar(inputJar, jsonShrinkingType, mappingsFile)
-		deflate(outputJar, deflateAlgorithm)
+//		if (useProguard)
```

Comment:
Any particular reason we're keeping these comments around?

Reply:
nope